### PR TITLE
add avatar-alert-badge

### PR DESCRIPTION
**Pull request by gauravsinhaweb**

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  background-color: #ff0000;
+}
+.idle {
+  background-color: #868686;
+}
+.av_badge_lg,
+.av_badge_md,
+.av_badge_sm {
+  position: absolute;
+  /* border-radius: 50%; */
```
Comment (iRohitGaur) [inline components/badge/badge.css]: remove the commented code

**Review by iRohitGaur — APPROVED** [review]
Great work! Good to merge.

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  background-color: #edf7ed;
+  color: #3d603f;
+}
+.alert-warning {
+  background-color: #fff4e5;
+  color: #764f17;
+}
+.alert-info {
+  background-color: #e5f6fd;
+  color: #326a83;
```
Comment (rdthetha) [inline components/alert/alert.css:24]: It would have been better to use variables for colors

Comment (gauravsinhaweb) [reply components/alert/alert.css:24]: Thanks for the review, will make changes.

File: components/badge/badge.css (hunk 1 above)
Comment (gauravsinhaweb) [reply components/badge/badge.css]: Thanks for the review, will resolve the changes

**Comment by j24m** [comment]
Great work Gaurav , good to merge:)

**Comment by manojsarna** [comment]
1. change the fontsize , color and padding for the mentioned classes in the paragraphs.

2. dont provide the margins for the individual component elements. Instead use (hint : flex )

**Review by manojsarna — COMMENTED** [review]
change the fontsize , color and padding for the mentioned classes in the paragraphs.

dont provide the margins for the individual component elements. Instead use (hint : flex )

**Pull request merged**